src/context.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react'
+import type { UnistylesBreakpoints } from './global'
+import { unistyles } from './core'
+import type {
+    ScreenDimensions,
+    ScreenInsets,
+    ScreenSize,
+    UnistylesEvents,
+    UnistylesMobileLayoutEvent,
+    UnistylesTheme,
+    UnistylesThemeEvent
+} from './types'
+import { UnistylesEventType } from './common'
+import { NativeEventEmitter, NativeModules } from 'react-native'
+
+export type TUnistylesContext = {
+    plugins: Array<string>;
+    theme: UnistylesTheme;
+    layout: {
+        screen: ScreenSize;
+        statusBar: ScreenDimensions;
+        navigationBar: ScreenDimensions;
+        insets: ScreenInsets;
+        breakpoint: keyof UnistylesBreakpoints;
+        orientation: 'landscape' | 'portrait';
```

Comment:
it also has a separate type in the core

Reply:
Nevermind, I need to add docs about it so I will fix it later.

The plan is to test it & release it this week (Somewhere around Friday)